stable/concourse/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: concourse
-version: 1.0.6
+version: 1.0.7
```

Comment:
You'll need to bump the chart version to 1.0.8 as another PR was merged before yours.